modules/widget-visibility/widget-conditions.php

```diff
@@ -45,7 +45,7 @@ public static function widget_admin_setup() {
 
 		$categories = get_categories(
 			array(
-				'number'  => 1000,
+				'number'  => apply_filters( 'jetpack_widget_visibility_max_number_categories', 1000 ),
```

Comment:
Could you add a docblock before each filter, following this format?
https://github.com/Automattic/jetpack/blob/984e44ca469599d7536cf228432fc80439ccbed9/modules/widget-visibility/widget-conditions.php#L145-L156

It will allow us to automatically generate a Codex page for each filter here:
http://developer.jetpack.com/

Reply:
Done in 75f45bcfa9c0e2901c6a8b56b6971b63def84a2c